src/Session/SessionStorage.php

```diff
@@ -353,9 +353,10 @@ protected function collectGarbage()
             return;
         }
         $divisor = $this->options->getInt('gc_divisor');
+        $probabilityPercentage = 100 * $probability / $divisor;
+        $chance = mt_rand(1, 100);
 
-        $rand = mt_rand(0, $divisor);
-        if ($rand < $probability) {
+        if ($chance <= $probabilityPercentage) {
```

Comment:
Also, I miust have copy/pasta-ed something … and this was firing given enough refreshes